web/styling_elements/applying_styles_by_piercing_shadow_dom_boundaries/index.html

```diff
@@ -3,19 +3,21 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <link rel="import" href="my_element.html">
-    <style shim-shadowdom>
-      body /deep/ h2 {
+    <script src="packages/web_components/webcomponents-lite.min.js"></script>
+    <script src="packages/browser/dart.js"></script>
+    <style is="custom-style">
+      :root {
```

Comment:
shouldn't this be `my-element` instead of `:root`? I haven't actually played much with the new styling stuff, so I could be wrong but this seems weird ;)